ci/cpu/build.sh

```diff
@@ -34,6 +34,9 @@ env
 logger "Activate conda env..."
 source activate gdf
 
+logger "Bump pyarrow"
+conda install -c conda-forge pyarrow=0.11.1 arrow-cpp=0.11.1 pandas>=0.23.4
```

Comment:
@raydouglass @mike-wendt Seeing as I need to bump these in the CI environment not sure if we want these in the scripts, will let you advise.

Reply:
This should be fine, I'll strip this out in the release PR